Packs/Akamai_SIEM/Integrations/Akamai_SIEM/README.md

```diff
@@ -1,383 +1,152 @@
+
+```json
+[
+    {
         "name": "Akamai SIEM: 50170",
         "occurred": "2019-12-10T18:28:27Z",
         "rawJSON": {
-          "type": "akamai_siem",
-          "format": "json",
-          "version": "1.0",
-          "attackData": {
-            "configId": "50170",
-            "policyId": "1234",
-            "clientIP": "8.8.8.8",
-            "rules": "test",
-            "ruleVersions": "",
-            "ruleMessages": "Test",
-            "ruleTags": "Test",
-            "ruleData": "",
-            "ruleSelectors": "",
-            "ruleActions": "Test"
-          },
-          "httpMessage": {
-            "requestId": "3fbce3e",
-            "start": "1576002507",
-            "protocol": "HTTP/1.1",
-            "method": "HEAD",
-            "host": "google.com",
-            "port": "80",
-            "path": "index",
-            "requestHeaders": "Test",
-            "status": "403",
-            "bytes": "0",
-            "responseHeaders": "Server"
-          },
-          "geo": {
-            "continent": "NA",
-            "country": "US",
-            "city": "LOSANGELES",
-            "regionCode": "CA",
-            "asn": "5650"
-          }
+            "type": "akamai_siem",
+            "format": "json",
+            "version": "1.0",
+            "attackData": {
+                "configId": "50170",
+                ...
+            }
         }
-      },
-      {
+    },
+    {
         "name": "Akamai SIEM: 50170",
         "occurred": "2019-12-10T18:28:26Z",
         "rawJSON": {
-          "type": "akamai_siem",
-          "format": "json",
-          "version": "1.0",
-          "attackData": {
-            "configId": "50170",
-            "policyId": "1234",
-            "clientIP": "8.8.8.8",
-            "rules": "test",
-            "ruleVersions": "",
-            "ruleMessages": "Test",
-            "ruleTags": "Test",
-            "ruleData": "",
-            "ruleSelectors": "",
-            "ruleActions": "Test"
-          },
-          "httpMessage": {
-            "requestId": "3fbd757",
-            "start": "1576002506",
-            "protocol": "HTTP/1.1",
-            "method": "HEAD",
-            "host": "google.com",
-            "port": "80",
-            "path": "index",
-            "requestHeaders": "Test",
-            "status": "403",
-            "bytes": "0",
-            "responseHeaders": "Server"
-          },
-          "geo": {
-            "continent": "NA",
-            "country": "US",
-            "city": "LOSANGELES",
-            "regionCode": "CA",
-            "asn": "5650"
-          }
+            "type": "akamai_siem",
+            "format": "json",
+            "version": "1.0",
+            "attackData": {
+                "configId": "50170",
+                ...
+            }
         }
-      }
-    ]
-</pre>
-
-<h2>Commands</h2>
-<p>
-    You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
```

Comment:
You can execute these commands from the CLI, as part of a script, or in a playbook.